packages/block/src/header.ts

```diff
@@ -518,13 +518,13 @@ export class BlockHeader {
       )
       throw new Error(msg)
     }
-    if ('cliqueActiveSigners' in blockchain === false) {
+    if ('cliqueActiveSigners' in (blockchain as any).consensus === false) {
       const msg = this._errorMsg(
         'PoA blockchain requires method blockchain.cliqueActiveSigners() to validate clique difficulty'
```

Comment:
```suggestion
        'PoA blockchain requires method blockchain.consensus.cliqueActiveSigners() to validate clique difficulty'
```

Reply:
Good catch - updated

Reply:
I think this was reverted, if you want to re-apply

Reply:
also for the line above it, is it possible to cast as CliqueConsensus instead of any? if not no worries